notNeededPackages.json

```
@@ -3614,6 +3614,12 @@
             "libraryName": "rc-progress",
             "asOfVersion": "2.4.0"
         },
+        {
+            "libraryName": "rdf-js",
+            "typingsPackageName": "rdf-js",
+            "asOfVersion": "2.0"
+        },
+        {
```

Guessing this should be

```suggestion
        "rdf-js": {
            "libraryName": "rdf-js",
            "asOfVersion": "2.0"
        },
```